TestTools/StreamChatTestTools/Mocks/StreamChat/ConnectionRepository_Mock.swift

```diff
@@ -0,0 +1,125 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+
+/// Mock implementation of `ChatClientUpdater`
+final class ConnectionRepository_Mock: ConnectionRepository, Spy {
```

Comment:
We have an issue here in SwiftUI - `ConnectionRepository` is not public and it breaks the tests.